ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -2,6 +2,92 @@
 // E2E Jenkins file.
 //
 
+import groovy.transform.Field
+
+@Field
+def podYAML = '''
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: main
+    image: gcr.io/k8s-testimages/kubekins-e2e:v20191108-9467d02-master
+    command:
+    - runner.sh
+    - sleep
+    - 99d
+    # we need privileged mode in order to do docker in docker
+    securityContext:
+      privileged: true
+    env:
+    - name: DOCKER_IN_DOCKER_ENABLED
+      value: "true"
+    resources:
+      requests:
+        memory: "8000Mi"
+        cpu: 8000m
+    # kind needs /lib/modules and cgroups from the host
+    volumeMounts:
+    - mountPath: /lib/modules
+      name: modules
+      readOnly: true
+    - mountPath: /sys/fs/cgroup
+      name: cgroup
+    # dind expects /var/lib/docker to be volume
+    - name: docker-root
+      mountPath: /var/lib/docker
+    # legacy docker path for cr.io/k8s-testimages/kubekins-e2e
+    - name: docker-graph
+      mountPath: /docker-graph
+  volumes:
+  - name: modules
+    hostPath:
+      path: /lib/modules
+      type: Directory
+  - name: cgroup
+    hostPath:
+      path: /sys/fs/cgroup
+      type: Directory
+  - name: docker-root
+    emptyDir: {}
+  - name: docker-graph
+    emptyDir: {}
+'''
```

Comment:
I suggest moving this pod template to the Jenkins [system configuration](https://internal.pingcap.net/idc-jenkins/configure) so that all jobs on Jenkins can use this template just like before.

Reply:
SRE team can copy this to system configuration for their own use. But I don't like to depend on global configurations. Different jobs have different requirements. Easier to update image and configurations (e.g. resources, volume configuration) and fewer dependencies (don't worry about affecting other jobs or being affected by changes in Jenkins configurations).

Reply:
And this makes it easier for people outside of our company to know how we run our e2e jobs.

Reply:
Make sense.